src/Connector.SqlServer/Features/DefaultBuildCreateIndexFeature.cs

```diff
@@ -12,21 +13,21 @@ public class DefaultBuildCreateIndexFeature : IBuildCreateIndexFeature
         public virtual IEnumerable<SqlServerConnectorCommand> BuildCreateIndexSql(
```

Comment:
Do not change the original function and mark as obsolete, to handle existing functionality/implementations.
Create an additional overload to accept the "schema".